docs/_guide/styles.md

```diff
@@ -220,9 +220,30 @@ There are some important caveats though:
 * External styles can cause a flash-of-unstyled-content (FOUC) while they load.
 * The URL in the `href` attribute is relative to the _main document_, making this technique mostly useful for application elements where asset URLs are well known, and not for reusable elements published publicly.
 
+Now, be sure what are you doing before do this: by the way, if you `really need` or ` really want` a `separate css file from main component file`, you can use the `unsafeCSS` function to import the "external css" like the example below.
```

Comment:
Why are "really need", "really want", and "separate css file from main component file" formatted as code?

Reply:
I wanted to put a little emphasis on these words